torch/_inductor/triton_heuristics.py

```diff
@@ -216,7 +216,11 @@ def launcher({', '.join(def_args)}, grid, stream):
                     grid_0, grid_1, grid_2 = grid(grid_meta)
                 else:
                     grid_0, grid_1, grid_2 = grid
-                bin.c_wrapper(grid_0, grid_1, grid_2, bin.num_warps, bin.shared,
+                # use default value for num_ctas and clusterDims
+                num_ctas = 1
+                clusterDims = [1, 1, 1]
```

Comment:
The Triton has defined the default value. I think we should use `bin.num_ctas`, `bin.clusterDims[0]`, `bin.clusterDims[1]`, `bin.clusterDims[2]` directly.

```python
bin.c_wrapper(grid_0, grid_1, grid_2, bin.num_warps,
               bin.num_ctas, bin.clusterDims[0], bin.clusterDims[1], bin.clusterDims[2],
               bin.shared, stream, bin.cu_function, None, None, None,
               {', '.join(call_args)})
```

Reply:
yea, good suggestion